apps/src/gamelab/blocks.js

```diff
@@ -46,14 +48,24 @@ const customInputTypes = {
       return block.getTitleValue(arg.name);
     },
   },
+  costumePicker: {
+    addInput(block, input) {
+      block.appendDummyInput()
+        .appendTitle(input.label)
+        .appendTitle(new Blockly.FieldDropdown(sprites), input.name);
+    },
+    generateCode(block, arg) {
+      return block.getTitleValue(arg.name);
+    },
+  },
 };
 
 export default {
   install(blockly, blockInstallOptions) {
     const SPRITE_TYPE = blockly.BlockValueType.SPRITE;
     const { ORDER_MEMBER, ORDER_ATOMIC } = Blockly.JavaScript;
 
-    const sprites = () => {
+    sprites = () => {
```

Comment:
I'd just move this whole function definition outside of `install()` instead of initializing `sprite` to an empty list and then updating it later.